user_guide_src/source/libraries/validation/046.php

```diff
@@ -16,7 +14,9 @@ public function process()
     {
         // ...
 
-        $validation = Services::validation();
+        $data = [];
+
```

Comment:
Why do you add `$data = [];`?
It means the empty array will be validated in:
https://github.com/codeigniter4/CodeIgniter4/blob/375f6f6123ca2cbbc544b174fdd38bde30c7f948/user_guide_src/source/libraries/validation/046.php#L37